src/audio/kpb.c

```diff
@@ -165,7 +165,7 @@ static struct comp_dev *kpb_new(struct sof_ipc_comp *comp)
 			       &ops, /* task ops */
 			       &kpb->draining_task_data, /* task private data */
 			       0, /* core on which we should run */
-			       SOF_SCHEDULE_FLAG_IDLE);
+			       0); /* no flags */
```

Comment:
Personally I'd rather not keep unused function parameters and struct fields unless there are *certain* plans to begin using them in the near future. Do you have anything in the pipeline to start using .flags? If not, maybe better remove them.

Reply:
There are specific plans to use flags e.g. to differentiate sporadic tasks.
Better to keep them here.